### PR TITLE
Change hi<->lo priority

### PR DESCRIPTION
**Pull request by gorryfair**
Suggested to align with other IETF specs. Closes 1378 ifhen merged.

### CODE REVIEW

**Comment by gorryfair** [comment]
No mention of the priority order was found in the arch and implementation specs.

File: draft-ietf-taps-interface.md [hunk 1]
```diff
@@ -3082,8 +3082,8 @@ at their defaults.
 
 The Transport Services API does order `connPriority` over
 `msgPriority`. In the absence of other externalities
-(e.g., transport-layer flow control), a priority 1 Message on a priority 0
-Connection will be sent before a priority 0 Message on a priority 1
+(e.g., transport-layer flow control), a priority 0 Message on a priority 1
```
Comment (mwelzl) [inline draft-ietf-taps-interface.md:3085]:
```suggestion
(e.g., transport-layer flow control), a priority 1 Message on a priority 0
```

File: draft-ietf-taps-interface.md [hunk 2]
```diff
@@ -3082,8 +3082,8 @@ at their defaults.
...
+Connection will be sent before a priority 1 Message on a priority 0
```
Comment (mwelzl) [inline draft-ietf-taps-interface.md:3086]:
```suggestion
Connection will be sent before a priority 0 Message on a priority 1
```

**Review by mwelzl — CHANGES_REQUESTED** [review]
There is an example that you changed, but I think it was right before. I think this is what happened:
Version 1 which we once had: this was written, and correct.
Version 2, where we flipped the order: this was overlooked, and therefore wrong.
This PRs version, where we flip the order again: now this previously-wrong text from version 2  automatically becomes correct again.

File: draft-ietf-taps-interface.md [hunk 3]
```diff
@@ -1930,8 +1930,8 @@ Connection -> CloneError<reason?>
 The `connPriority` Connection Property operates on Connections in a Connection Group
 using the same approach as in {{msg-priority}}: when allocating available network
 capacity among Connections in a Connection Group, sends on Connections with
-higher Priority values will be prioritized over sends on Connections that have
-lower Priority values. Capacity will be shared among these Connections according to
+lower Priority values will be prioritized over sends on Connections that have
+higher Priority values. Capacity will be shared among these Connections according to
```
Comment (tfpauly) [inline draft-ietf-taps-interface.md:1934]:
```suggestion
numerically lower Priority values will be prioritized over sends on Connections that have
numerically  higher Priority values. Capacity will be shared among these Connections according to
```

File: draft-ietf-taps-interface.md [hunk 4]
```diff
@@ -2096,7 +2096,7 @@ Default:
 This property is a non-negative integer representing the
 priority of this Connection
 relative to other Connections in the same
-Connection Group. A higher value reflects a higher priority. It has no effect
+Connection Group. A lower value reflects a higher priority. It has no effect
```
Comment (tfpauly) [inline draft-ietf-taps-interface.md:2099]:
```suggestion
Connection Group. A numerically lower value reflects a higher priority. It has no effect
```

File: draft-ietf-taps-interface.md [hunk 5]
```diff
@@ -2685,10 +2685,10 @@ Default:
 : 100
 
 This property specifies the priority of a Message, relative to other Messages sent over the
-same Connection.
+same Connection. A lower value represents a higher priority.
```
Comment (tfpauly) [inline draft-ietf-taps-interface.md:2688]:
```suggestion
same Connection. A numerically lower value represents a higher priority.
```

**Review by tfpauly — CHANGES_REQUESTED** [review]

**Comment by mwelzl** [comment]
@tfpauly - great suggestions!  I felt uneasy about these "lower" / "higher" words but didn't quite know what to do... I think putting "numerically" there makes it super clear.

File: draft-ietf-taps-interface.md [hunk 6]
```diff
@@ -1930,8 +1930,8 @@ Connection -> CloneError<reason?>
 The `connPriority` Connection Property operates on Connections in a Connection Group
 using the same approach as in {{msg-priority}}: when allocating available network
 capacity among Connections in a Connection Group, sends on Connections with
-higher Priority values will be prioritized over sends on Connections that have
-lower Priority values. Capacity will be shared among these Connections according to
+numerically lower Priority values will be prioritized over sends on Connections that have
+numerically  higher Priority values. Capacity will be shared among these Connections according to
```
Comment (mwelzl) [inline draft-ietf-taps-interface.md:1934]:
```suggestion
numerically higher Priority values. Capacity will be shared among these Connections according to
```

**Review by tfpauly — APPROVED** [review]

**Pull request merged**